src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -463,6 +463,17 @@ public int compare(Goal a, Goal b) {
           }
         }
       }
+      if (Config.SidebarMessage.enabled()) {
+        if (Config.SidebarMessage.atTop()) {
+          rows.add(0, Config.SidebarMessage.format());
+        } else {
+          if (rows.size() >= MAX_ROWS) {
+            rows.set(MAX_ROWS - 1, Config.SidebarMessage.format());
+          } else {
+            rows.add(Config.SidebarMessage.format());
+          }
+        }
+      }
```

Comment:
Can we have a player setting to disable scoreboard messages?

Reply:
This wouldn't be possible with the way sidebars are currently rendered. The sidebar doesn't know which player is viewing it, only the player's team.

Reply:
Don't add the setting then lol ;p
https://github.com/Electroid/PGM/pull/176/commits/98246c32928f7490406181d41dd0c4a0d7004323#diff-f08f8cdb20626aaae1393d6a6ea2a79fR13-R16